modules/creating-cluster-with-aws-kms-key.adoc

```diff
@@ -0,0 +1,211 @@
+// Module included in the following assemblies:
+//
+// * monitoring/enabling-monitoring-for-user-defined-projects.adoc
+
+:_content-type: PROCEDURE
+[id="creating-cluster-with-aws-kms-key"]
+= Creating a ROSA cluster using a custom AWS KMS key
+
+You can create a {product-title} (ROSA) cluster with a customer-provided KMS key that is used to encrypt both node root volumes and persistent volumes for mounted EBS claims.
```

Comment:
@fmcdonal HCP has 2 `rosa` CLI flags related to customer KMS encryption

- `--enable-customer-managed-key` along with `--kms-key-arn` will encrypt a node's _root volume *only*_ (this is different from ROSA Classic where it _will_ encrypt both the root volume and default `StorageClass`)
- `--etcd-encryption-kms-arn`, this will encrypt the customer's ETCD database in the ROSA Service Account with the customer's specified KMS key

Does this help clarify?

Reply:
```suggestion
You can create a {product-title} (ROSA) cluster with a customer-provided KMS key that is used to encrypt either node root volumes, the etcd database, or both. A different KMS key ARN can be provided for each option.

[NOTE]
====
{hcp-title} does not automatically configure the `default` StorageClass to encrypt persistent volumes with the customer-provided KMS key. This is something that can be configured in-cluster after installation.
====
```